launcher/image/preload.sh

```diff
@@ -6,6 +6,8 @@ readonly EXPERIMENTS_BINARY="confidential_space_experiments"
 readonly GPU_REF_VALUES_PATH="${CS_PATH}/gpu"
 readonly COS_GPU_INSTALLER_IMAGE_REF="${GPU_REF_VALUES_PATH}/cos_gpu_installer_image_ref"
 readonly COS_GPU_INSTALLER_IMAGE_DIGEST="${GPU_REF_VALUES_PATH}/cos_gpu_installer_image_digest"
+readonly DRIVER_DIGEST_SHA256SUM="${GPU_REF_VALUES_PATH}/driver_digest_sha256sum"
```

Comment:
nit: `DRIVER_DIGEST_SHA256SUM`/
for consistency, either make the above `COS_GPU_INSTALLER_IMAGE_DIGEST_SHA256` or the below `DRIVER_DIGEST`

Reply:
Same with the file names